packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -309,6 +309,30 @@ void main() {
       expect(count, 6);
     });
 
+    testWidgets('pumpContinuous', (WidgetTester tester) async {
+      await tester.pumpWidget(const Text('foo', textDirection: TextDirection.ltr));
+      int count;
+
+      final AnimationController test = AnimationController(
+        duration: const Duration(milliseconds: 200),
+        vsync: tester,
+      );
+      count = await tester.pumpContinuous(const Duration(milliseconds: 80));
+      expect(count, 1); // No animation, it always pumps at least one frame
+
+      test.forward(from: 0.0);
+      count = await tester.pumpContinuous(const Duration(milliseconds: 80));
+      // ends after 80 ms
+      expect(count, 1 + (80E3/kDefaultFrameIntervalInMicroseconds).floor());
```

Comment:
This and the expect below seem very confusing to write and easy to mess up, and easy to end up with rounding issues that will throw a test off. Why aren't we just expectcing the number of frames we want explicitly?

Reply:
Because I want to give the reasoning of the number of frames. `1 + (80E3/kDefaultFrameIntervalInMicroseconds).floor()` is consistent with what I wrote in dartdoc of the method, and the reason I wrote `test.forward(from: 0.4);` is to avoid rounding error.